src/universal/components/Masthead/DefaultMasthead/DefaultMasthead.mobile.tsx

```diff
@@ -25,21 +26,30 @@ const Logo = styled.span`
 const DefaultMasthead = ({
   query,
   handleChangeQuery,
-  handleClickLeftArrow,
   handleClickHamburger,
+  handleClickLeftArrow,
   handleClickMarmoymLogo,
   handleClickPencil,
   handleClickSearch,
   handleKeyDown,
+  isSearchState,
   modalIsVisible,
   searchRequested,
 }) => {
   return (
     <MastheadBase>
       <ButtonGroup>
-        <Button onClick={handleClickMarmoymLogo}>
+        {isSearchState() ? (
```

Comment:
usually when we code, `-isXXX` is data of type boolean. Could we pass this as either `true/false ?